tests/playwright/sanity/modules/promotions/admin-menu-items/menu-item-promotions.test.ts

```diff
@@ -1,26 +1,61 @@
-import { test, expect, type Page } from '@playwright/test';
+// @ts-ignore
+
+import { test, expect } from '@playwright/test';
 import WpAdminPage from '../../../../pages/wp-admin-page';
 
-test( 'Promotion screenshots', async ( { page }, testInfo ) => {
-	const wpAdminPage = new WpAdminPage( page, testInfo ),
-		promotionContainer = '.e-feature-promotion';
+test.describe( 'Promotion tests @promotions', () => {
+	test( 'Menu Items Promotions - screenshots', async ( { page }, testInfo ) => {
+		const wpAdminPage = new WpAdminPage( page, testInfo ),
+			promotionContainer = '.e-feature-promotion';
 
-	await wpAdminPage.login();
+		await wpAdminPage.login();
 
-	await test.step( 'Free to Pro - Submissions', async () => {
-		await wpAdminPage.promotionPageScreenshotTest( promotionContainer, 'e-form-submissions', 'submissions-menu-item-desktop' );
-	} );
+		await test.step( 'Free to Pro - Submissions', async () => {
```

Comment:
I'm not a fan of this `test.step`. If you are using page object and wrap playwright code in functions, then it is readable and makes no sense in additional wrappers

Reply:
But this is up to you to leave it or remove

Reply:
I think I will leave it, just to increase the readability